spec/scope_resolver_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "spec_helper"
+
+module Pundit
+  describe ScopeResolver do
+    subject { described_class.new(user, resource_scope) }
+    let(:user) { double('pundit_user') }
+
+    describe '#resolve' do
+      context 'policy class cannot be found' do
+        let(:resource_scope) { nil }
+
+        it { expect(subject.resolve).to be nil }
```

Comment:
This one-liner syntax works best when making statements directly about the subject (rather than methods called on the subject), which allows use of `is_expected` (see [docs](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax)).

As it is, the subject feels like a bit of a [mystery guest](https://robots.thoughtbot.com/mystery-guest).

Would these tests be clearer if the `subject` and `before` were inlined? Are we getting a benefit from extracting the setup of the tests into `let` and `before`?